src/main/java/org/fhirvp/websocket/FHIRNotificationClient.java

```diff
@@ -0,0 +1,31 @@
+package org.fhirvp.websocket;
+
+import io.quarkus.runtime.StartupEvent;
+import org.fhirvp.websocket.context.FHIRNotificationConfig;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import javax.websocket.ContainerProvider;
+import javax.websocket.DeploymentException;
+import javax.websocket.Session;
+import java.io.IOException;
+import java.net.URI;
+
+@ApplicationScoped
+public class FHIRNotificationClient {
+
+    Session fhirNotificationClientSession;
+
+    @Inject
+    FHIRNotificationConfig config;
+
+    public void onStart(@Observes StartupEvent event) {
+        try {
+            fhirNotificationClientSession = ContainerProvider.getWebSocketContainer().connectToServer(FHIRNotificationClientEndpoint.class, URI.create(config.websocket().url()));
+        } catch (DeploymentException | IOException e) {
+            e.printStackTrace();
```

Comment:
IMO this can just throw, if we have a problem establishing a connection it points to misconfiguration of the server, in which case it cannot fulfil its intended role. We offer no other functionality that does not depend on the FHIR server.

Reply:
Addressed at 54ec843